pepper-apis/studybuilder-cli/studies/lms/patches/consent-version2.conf

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Can we rename this file as  consent-pdf-version2.conf so that we know its  pdf config.